src/states/node.rs

```diff
@@ -842,8 +842,8 @@ impl Node {
                             dst: Authority::NaeManager(prefix.lower_bound()),
                             content: MessageContent::GroupSplit(our_group_prefix),
                         };
-                        // We send on route 1 by default to avoid trying to send via the newly joined
-                        // node
+                        // We send on route 1 by default to avoid trying to
+                        // send via the newly joined node
```

Comment:
I'm pretty sure I made that change because `rustfmt` _did_ complain about it. Maybe it doesn't for you; I run it manually on the entire project, not just the files I change.